docs/setup/settings.asciidoc

```diff
@@ -455,6 +455,13 @@ manner that is inconsistent with `/proc/self/cgroup`.
  | Override for cgroup cpuacct path when mounted
 in a manner that is inconsistent with `/proc/self/cgroup`.
 
+| `savedObjects.maxImportExportSize:`
```

Comment:
Could you add documentation for `maxImportPayloadBytes` too? If you're exporting more, you will more likely bump into that limit when importing.

I think it will help users if we mention these settings from the import/export and management docs:
 - https://github.com/elastic/kibana/blob/master/docs/api/saved-objects/export.asciidoc
 - https://github.com/elastic/kibana/blob/master/docs/api/saved-objects/import.asciidoc
 - https://github.com/elastic/kibana/blob/master/docs/management/managing-saved-objects.asciidoc#L38

Reply:
good idea, added all of the above.